beispiele/annahme-mit-vollstaendigen-angaben-ein-antragsteller.md

```diff
@@ -423,7 +423,11 @@ Anmerkung zum Tilgungsplan:
         }
       }
     },
-    "maximalerAuszahlungsbetrag": null
+    "maximalerAuszahlungsbetrag": null,
+    "alternative": {
```

Comment:
hier wäre ein eigenes Downsell Beispiel besser um klarer zu machen, dass dieser Knoten nur in diesem Fall zu befüllen ist